### PR TITLE
chore: update apecloud-mysql binary version

### PR DESCRIPTION
**Pull request by sophon-zt**

### CODE REVIEW

File: addons/apecloud-mysql/config/mysql8-config-constraint.cue [hunk 1]
```diff
@@ -1536,22 +1536,40 @@
...
+	// The option to enable or disable audit log.
+	audit_log_enabled?: string & "0" | "1" | "OFF" | "ON" | *"OFF"
+
+	// The policy controlling how the audit log plugin writes connection events to its log file. Supported values are 'ALL' (default), 'ERRORS' and 'NONE'.
+	audit_log_connection_policy?: string & "ALL" | "ERRORS" | "NONE"
+
+	// The policy controlling how the audit log plugin writes  query events to its log file. Supported values are 'ALL' (default), 'UPDATES',  'UPDATES_OR_ERRORS', 'ERRORS' and 'NONE'.
+	audit_log_statement_policy?: string & "ALL" | "UPDATES" | "NONE" | "ERRORS" | "UPDATES_OR_ERRORS"
+
+	// Max number of rows in each audit log file. Log records will be discarded above this number.
```
Comment (demian0110) [inline addons/apecloud-mysql/config/mysql8-config-constraint.cue:1571]: duplicated with line 1554

File: addons/apecloud-mysql/config/mysql8-config-constraint.cue [hunk 2]
```diff
@@ -1536,22 +1536,40 @@
...
 	// The policy controlling how the audit log plugin writes  query events to its log file. Supported values are 'ALL' (default), 'UPDATES',  'UPDATES_OR_ERRORS', 'ERRORS' and 'NONE'.
-	rds_audit_log_statement_policy?: string & "ALL" | "UPDATES" | "NONE" | "ERRORS" | "UPDATES_OR_ERRORS"
+	loose_audit_log_statement_policy?: string & "ALL" | "UPDATES" | "NONE" | "ERRORS" | "UPDATES_OR_ERRORS"
 
 	// Max number of rows in each audit log file. Log records will be discarded above this number.
-	rds_audit_log_row_limit?: int & >=1 | *100000
+	audit_log_row_limit?: int & >=1 | *100000
+
+	// Mysql audit log version.
+	audit_log_version?: string & "MYSQL_V1" | "MYSQL_V3"
```
Comment (demian0110) [inline addons/apecloud-mysql/config/mysql8-config-constraint.cue:1557]: why not with the prefix loose_?

Comment (sophon-zt) [reply addons/apecloud-mysql/config/mysql8-config-constraint.cue:1557]: The mysql engine itself accepts both types of parameters. In fact, it is not sure which set of parameters will be used by the user-defined template. so it retains maximum consistency with the engine.

**Review by heng4fun — APPROVED** [review]

**Pull request merged**